axelor-account/src/main/resources/views/ReconcileGroup.xml

```diff
@@ -24,13 +23,16 @@
         </viewer>
       </field>
     </panel>
-    <panel name="company" showTitle="false">
+    <panel name="company" showTitle="false" readonly="true">
       <field name="company"/>
     </panel>
-    <panel name="main" showIf="company">
+    <panel name="main" showIf="company" readonly="true">
       <panel-dashlet action="action-reconcile-group-view-reconcile" colSpan="12"/>
       <panel-dashlet action="action-reconcile-group-view-move-line" colSpan="12"/>
     </panel>
+    <panel sidebar="true">
+      <button name="cancel" title="Cancel" onClick="action-reconcile-group-method-unreconcile"/>
```

Comment:
Change button name and title as 'unreconcile' and 'Unreconcile', change action name accordingly.